oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/oc/OCMetricHandler.java

```diff
@@ -107,6 +118,13 @@
         return result;
     }
 
+    private static Map<String, String> buildLabelsFromNodeInfo(Node node,
+                                                               Map<String, String> buildLabelsResult) {
+        if (node != null)
+            buildLabelsResult.put("node_identifier_host_name", node.getIdentifier().getHostName());
```

Comment:
Could we add other fields more than `hostname` in `ProcessIdenifier`?

Reply:
```
// Identifier that uniquely identifies a process within a VM/container.
message ProcessIdentifier {

  // The host name. Usually refers to the machine/container name.
  // For example: os.Hostname() in Go, socket.gethostname() in Python.
  string host_name = 1;

  // Process id.
  uint32 pid = 2;

  // Start time of this ProcessIdentifier. Represented in epoch time.
  google.protobuf.Timestamp start_timestamp = 3;
}
```

I think there is only `pid` makes sense? Do you think `_` is good, or should consider `.`? Such as `node.identifier.host_name`.

Reply:
MAL has a capacity to transform `.` to `_`, which means no matter what we choose, the final result must be `node_identifier_host_name `.

Reply:
OK, then `_` is fine.

Reply:
A common use case about the `start_timestamp` is to get `uptime` metric, the expression should leverage `start_timestamp` and `MAL` 's `now()` function as below:

`now()-start_timestamp`

Reply:
I am just wondering whether this is the meaning, as it is reported every time, rather than a separated metric.